src/nvidia-cuda/devcontainer-feature.json

```diff
@@ -77,9 +77,10 @@
         "9.2.0.82",
         "9.2.1.18",
         "9.3.0.75",
-        "9.4.0.58"
+        "9.4.0.58",
+        "automatic"
```

Comment:
Let's move this value at the top of the list

Reply:
Done